src/seedu/addressbook/AddressBook.java

```diff
@@ -579,6 +585,21 @@ private static String executeListAllPersonsInAddressBook() {
         return getMessageForPersonsDisplayedSummary(toBeDisplayed);
     }
 
+    /**
+     * Requests confirmation from user
+     *
+     * @param message Command description for user prompt
+     * @return user's confirmation
+     */
+    private static boolean requestConfirmation(String message) {
+        showToUser(message + " " + MESSAGE_CONFIRMATION_PROMPT);
+        String userInput = getUserInput().replaceAll("\\s","");
+        echoUserCommand(userInput);
+        if(userInput.equals(COMMAND_CONFIRMATION))
```

Comment:
According to the [style guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used), you always need to have braces, even if it is a single line.

And there should always be a space between the `if` and the opening parenthesis.